EasyPost.NetFramework/Shipment.cs

```diff
@@ -300,14 +300,14 @@ public void Refund()
         ///     All invalid keys will be ignored.
         /// </param>
         /// <returns>Instance of EasyPost.ShipmentList.</returns>
```

Comment:
update the comment

Reply:
Resolved in [73a3961](https://github.com/EasyPost/easypost-csharp/pull/223/commits/73a39618c0e40963cd3dd1b750ef53a13e56f3d1)